src/convertToObject.js

```diff
@@ -1,12 +1,21 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
```

Comment:
u can do it immediately in ur case
```suggestion
  return sourceString
```